### PR TITLE
Secpol for auth-4.1.5, auth-4.0.6, rec-4.1.5 and rec-4.0.9

### PR DESCRIPTION
**Pull request by aerique**
Only merge this one when the repositories have been updated.

This PR also includes updated changelogs, new security advisories and the fixes for `master`.

### CODE REVIEW

File: docs/changelog/4.0.rst [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+
+Released 6th of November 2018
+
+This release fixes PowerDNS Security Advisory
+:doc:`2018-03 <../security-advisories/powerdns-advisory-2018-03>`: Crafted zone record can cause a denial of service (CVE-2018-10851)
+
+Bug fixes
+~~~~~~~~~
+
+- `#XXXX <https://github.com/PowerDNS/pdns/pull/XXXX>`__: Crafted zone record can cause a denial of service (CVE-2018-10851)
```
Comment (Habbie) [inline docs/changelog/4.0.rst]: XXXX?

File: docs/changelog/4.0.rst [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+- `#6013 <https://github.com/PowerDNS/pdns/pull/6013>`__: Skip v6-dependent test when pdns_test_no_ipv6 is set in environment
```
Comment (Habbie) [inline docs/changelog/4.0.rst]: This does not need to be in here.

File: docs/changelog/4.1.rst [hunk 3]
```diff
@@ -1,6 +1,61 @@
...
+  :released: November 6th 2018
+
+  This release fixes the following security advisories:
+
+  - PowerDNS Security Advisory :doc:`2018-03 <../security-advisories/powerdns-advisory-2018-03>` (CVE-2018-10851)
+  - PowerDNS Security Advisory :doc:`2018-05 <../security-advisories/powerdns-advisory-2018-05>` (CVE-2018-14626)
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: XXXX
```
Comment (Habbie) [inline docs/changelog/4.1.rst]: XXXX? I won't comment on all of them, there's at least one more.

File: docs/changelog/4.0.rst (hunk 2 above)
Comment (aerique) [reply docs/changelog/4.0.rst]: Assuming this referred just to the line with #6013.

Comment (Habbie) [reply docs/changelog/4.0.rst]: yes indeed!

**Pull request merged**